src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -52,9 +57,28 @@ public PersonCard(Person person, int displayedIndex) {
         name.setText(person.getName().value);
         patientId.setText("ID: " + person.getId().value);
         ward.setText("Ward: " + person.getWard().value);
+        /*
         diagnosis.setText("Diagnosis: " + person.getDiagnosis().value);
         medication.setText("Medication: " + person.getMedication().value);
-        notes.setText(person.getNotes().value);
+        notes.setText("Notes: " + (person.getNotes().toString().isEmpty() ? "-" : person.getNotes().value));
+        */
+        // Law of Demeter is not maintained, however the purpose simply retrieves information through getters,
+        // and there may be zero risk of modifications.
+        if (person.getAppointment() != null) {
```

Comment:
Just a nit, could be extracted out into a method to improve reusability

Reply:
I agree! Have extracted it out accordingly to enhance SLAP 👍